certbot_pkcs12.py

```diff
@@ -11,20 +13,19 @@ def _load_bytes(path):
 
 
 def _load_key(path):
-    return crypto.load_privatekey(crypto.FILETYPE_PEM, _load_bytes(path))
+    return load_pem_private_key(_load_bytes(path), password=None)
 
 
 def _load_cert(path):
-    return crypto.load_certificate(crypto.FILETYPE_PEM, _load_bytes(path))
+    return load_pem_x509_certificate(_load_bytes(path))
 
 
 def _load_certs(path):
     delimiter = b'-----BEGIN CERTIFICATE-----\n'
-    for section in _load_bytes(path).split(delimiter):
-        section = section.strip()
-        if section:
-            yield crypto.load_certificate(
-                crypto.FILETYPE_PEM, delimiter + section)
+     for section in _load_bytes(path).split(delimiter):
+         section = section.strip()
+         if section:
+             yield load_pem_x509_certificate(delimiter + section)
```

Comment:
I get `IndentationError: unexpected indent` here.

```suggestion
    for section in _load_bytes(path).split(delimiter):
        section = section.strip()
        if section:
            yield load_pem_x509_certificate(delimiter + section)
```